tests/test_dev_server.py

```diff
@@ -12,7 +14,7 @@
 
 class HelloWorld(Resource):
     @add_route("/", method="GET")
-    def list(self) -> Response:
+    def handler(self) -> Response:
```

Comment:
`handler()` is not the best name when we have `get()` declared below 😉 

I would leave the previous version as it harms no one but if you want to improve these endpoints, apply changes to the whole `HelloWorld` class.